### PR TITLE
feat: #1 added my personal team page part

### PR DESCRIPTION
**Pull request by jonathanschlue**
Resolves \#1

### Prerequisites

* [ ] Did you run your change locally?
* [ ] Are all issue todos really done?

For more information, see the style guide.

**Expected behavior:** page shows up with my personal content.

**Actual behavior:** it actually does on my browser locally.

### CODE REVIEW

File: public/team.html [hunk 1]
```diff
@@ -24,6 +24,13 @@
       <h5>Age: 28</h5>
       <p>I'm obsessed with cheese pizza and made a website for it!!!</p>
     </div>
+    <div class="member">
+      <h3>Jonathan Schlue</h3>
+      <img src="https://avatars2.githubusercontent.com/u/15818147?s=460&v=4" alt="Jonathan Schlue's profile picture">
```
Comment (danmcatee) [inline public/team.html]: Pls escape the semicolon in the alt tag

Comment (danmcatee) [reply public/team.html]: suggest 
&#39;

Comment (danmcatee) [reply public/team.html]: suggest \&#39;

Comment (jonathanschlue) [reply public/team.html]: I bet you ment the back tick. I'll fix it anyway.

**Pull request merged**